CHANGELOG.md

```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.6.0
+### Added
+* Service: Add Function App Settings to Bicep Template and deploy.py. [#1973](https://github.com/microsoft/onefuzz/pull/1973)
+* Agent: Add a timestamp to Agent log to make it easier to correlate events. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
+
+### Changed
+* Agent/Supervisor/Proxy: Rename the supervisor process from `onefuzz-supervisor` to `onefuzz-agent`. [#1989](https://github.com/microsoft/onefuzz/pull/1989)
```

Comment:
Note for future changelogs: we longer have a "supervisor". We should probably start referring to the new `onefuzz-agent` as "the agent" and `onefuzz-task` as the task worker / executor.

Reply:
Shall I make the edit for this round? or wait until next release?

Reply:
Next is fine! 👍